native-sql-engine/core/src/main/scala/com/intel/oap/expression/ConverterUtils.scala

```diff
@@ -517,6 +516,27 @@ object ConverterUtils extends Logging {
       throw new UnsupportedOperationException(s"Unsupported data type: $dt")
   }
 
+  def checkIfNestTypeSupported(dt: DataType): Unit = dt match {
+    case d: ArrayType => checkIfTypeSupported(d.elementType)
```

Comment:
Array[Array[]] still supported?

Reply:
checkIfTypeSupported is another check function, it has excluded the Array/Struct/Map  support.

Reply:
maybe rename this like checkPrimitiveTypes?

Reply:
"checkIfTypeSupported" is reuse the existed func.  It has been used in too many Operator scala files.